pollux/vc-jwt/src/test/scala/io/iohk/atala/pollux/vc/jwt/revocation/VCStatusList2021Spec.scala

```diff
@@ -13,7 +13,7 @@ object VCStatusList2021Spec extends ZIOSpecDefault {
 
   private def generateIssuer(): UIO[Issuer] = {
     val keyGen = KeyPairGenerator.getInstance("EC")
-    val ecSpec = ECGenParameterSpec("secp256r1")
+    val ecSpec = ECGenParameterSpec("secp256k1")
```

Comment:
The underlying signer uses ES256 JWT algo but we give it a `secp256k1` private key. Although the test works, it doesn't seems like an intended use of the JWT library. Should we just completely get rid of the `ES256Signer` since our crypto primitives doesn't support P-256 curve anyway?

Reply:
I also see the failing test after switching to `ES256KSigner` because the `JwtCirce`  doesn't support this algo natively. I traced back [where we do verification](https://github.com/hyperledger/identus-cloud-agent/blob/main/pollux/vc-jwt/src/main/scala/io/iohk/atala/pollux/vc/jwt/JWTVerification.scala#L86) and it looks like we use `JwtCirce` just to decode without signature option enabled and then validate the signature using pollux code. Probably we need to figure the right `validateXyz` function to call. We have loads of them :smile: and some have different `JwtOptions` (not sure why?).